CHANGELOG.md

```diff
@@ -3,8 +3,27 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [UNRELEASED]
+
+## 2022-10
+
+Added Escaped Ampersands Checker
```

Comment:
This should be removed - or added as prefix below `### Added`

There is no text after the release date